ocs_ci/ocs/constants.py

```diff
@@ -1142,3 +1142,20 @@
 MCG_NS_RESOURCE = 'ns_resource'
 MCG_NS_BUCKET = 'ns-bucket'
 MCG_NS_AWS_CONNECTION = 'aws_connection'
+
+# Create your own squads.
+# Squads are separated by Python packages in your project so it is important to
+# put the squad name in '.' --> ".squad_name." For example: In case this test
+# fails - tests.e2e.registry.test_pod_from_registry.TestRegistryImage.test_run_pod_local_image
+# the squad is ".registry." as can be seen in the Magenta squad below
```

Comment:
below SQUADS dict is not using ".squad_name" or ".registry". Can we remove or modify the comment accordingly

Reply:
Good point! Fixed by 8c87c59.